time_tracker_api/sql_repository.py

```diff
@@ -27,20 +29,15 @@ def init_app(app: Flask) -> None:
     db = SQLAlchemy(app)
 
     global SQLModel
-
-    class SQLModelClass(DatabaseModel):
-        def to_dto(self):
-            return self
-
-    SQLModel = SQLModelClass
+    SQLModel = DatabaseModel
```

Comment:
Checking at changes of `DatabaseModel` Class, and tracing use of `SQLModel` It seems to me that these two classes are not useful anymore.

Reply:
Whatever particularity that has to be inherited by the SQLModels should be done throw the SQL model. The `DatabaseModel` is platform-agnostic. Before we had this `to_dto` function that was required by such models. This is basically an encapsulation.